### PR TITLE
Fix:2698 Parse problem list and generate json file

### PR DESCRIPTION
**Pull request by deepak-gittyup**
Python script to parse exclude/ProblemList*.txt files and generate a unified ProblemList.json

Signed-off-by: deepak <deepak.gittyup@gmail.com>

### CODE REVIEW

**Comment by llxia** [comment]
Thanks @deepak-gittyup for working on this!

**Comment by llxia** [comment]
Could you also provide the output json file? Thanks

File: disabledTestParser/generateDisabledTestListJson.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    "linux-aarch64": ["aarch64_linux"],
+    "linux-arm": ["arm_linux"],
+    "linux-ppc64le": ["ppc64le_linux"],
+    "linux-s390x": ["s390x_linux"],
+    "linux-x64": ["x86-64_linux"],
+    "macosx-x64": ["x86-64_mac", "aarch64_mac"],
+    "windows-x64": ["x86-64_windows"],
+    "windows-x86": ["x86-64_windows", "x86-32_windows"],
+    "x86-64_windows": ["x86-64_windows"],
+    "z/OS-s390x": ["s390x_zos", "s390x_linux"]
```
[the remark this answers is not in the document]
Comment (deepak-gittyup) [reply disabledTestParser/generateDisabledTestListJson.py]: Addressed this issue

File: disabledTestParser/generateDisabledTestListJson.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+
+def get_jdk_version_and_impl(exclude_list_file):
+    # As of now, the ProblemList*.txt files are named in the following format:
+    # ProblemList_openjdk<JDK_VERSION >-<JDK_IMPL>.txt  # for JDK_IMPL = openj9 and sap
+    # or
+    # ProblemList_openjdk<JDK_VERSION>.txt              # for JDK_IMPL = hotspot ?
+    temp = exclude_list_file.replace("ProblemList_openjdk", "") \
+        .replace(".txt", "") \
+        .split("-")
```
[the remark this answers is not in the document]
Comment (deepak-gittyup) [reply disabledTestParser/generateDisabledTestListJson.py]: Done

**Comment by deepak-gittyup** [comment]
> Could you also provide the output json file? Thanks

Added the output ProblemList.json in the latest commit

Hi, I also see 6 others commits appearing here 
Looks like they were added between my first commit and my latest commit
Is this okay ?
Or should I close this one and send a new and clean PR ?

**Comment by llxia** [comment]
- Thanks @deepak-gittyup for ProblemList.json. Sorry, my msg was not very clear. I meant to see the output in the comment. I do not think we need to deliver it in the repo atm.

- This PR should not contain other people's commits. We do not need to close this one and create a new one. If you fix your local branch, you can force push to your remote branch.

- Please note, we may have a merge conflict with https://github.com/adoptium/aqa-tests/pull/2882. If https://github.com/adoptium/aqa-tests/pull/2882 is merged first, then this PR may need to rebase.

**Comment by deepak-gittyup** [comment]
> Thanks @deepak-gittyup for ProblemList.json. Sorry, my msg was not very clear. I meant to see the output in the comment. I do not think we need to deliver it in the repo atm.

I have removed ProblemList.json as a part fo the latest commit
Here is a sample output:
```
[
    {
        "TARGET": "jdk_custom",
        "CUSTOM_TARGET": "java/lang/Class/GetModuleTest.java",
        "GIT_ISSUE": "https://github.com/adoptium/aqa-tests/issues/1297",
        "PLATFORM": "all",
        "JDK_VERSION": "13",
        "JDK_IMPL": "openj9"
    },
    {
        "TARGET": "jdk_custom",
        "CUSTOM_TARGET": "java/lang/Class/GetPackageBootLoaderChildLayer.java",
        "GIT_ISSUE": "https://github.com/eclipse-openj9/openj9/issues/5274",
        "PLATFORM": "all",
        "JDK_VERSION": "13",
        "JDK_IMPL": "openj9"
    },
    {
        "TARGET": "jdk_custom",
        "CUSTOM_TARGET": "java/lang/Class/forName/NonJavaNames.sh",
        "GIT_ISSUE": "https://github.com/eclipse-openj9/openj9/issues/5225",
        "PLATFORM": "all",
        "JDK_VERSION": "13",
        "JDK_IMPL": "openj9"
    },
   .  .  . 
   .  .  . 
   .  .  . 
    {
        "TARGET": "jdk_custom",
        "CUSTOM_TARGET": "compiler/graalunit/UtilTest.java",
        "GIT_ISSUE": "https://github.com/adoptium/aqa-tests/issues/2804",
        "PLATFORM": "all",
        "JDK_VERSION": "11",
        "JDK_IMPL": "hotspot"
    },
    {
        "TARGET": "jdk_custom",
        "CUSTOM_TARGET": "javax/imageio/plugins/shared/ImageWriterCompressionTest.java",
        "GIT_ISSUE": "https://github.com/adoptium/aqa-tests/issues/2814",
        "PLATFORM": "ppc32_aix,ppc64_aix",
        "JDK_VERSION": "11",
        "JDK_IMPL": "hotspot"
    }
]
```

> This PR should not contain other people's commits. We do not need to close this one and create a new one. If you fix your local branch, you can force push to your remote branch.


I am still slowly getting used to git
But, I did not know that I could overwrite the previous commit history by doing force push
Thanks for letting me know about this 👍 

Are things looking okay now ?

**Review by llxia — APPROVED** [review]
LGTM

**Pull request merged**